datashare-tasks/src/main/java/org/icij/datashare/asynctasks/TaskManager.java

```diff
@@ -83,17 +73,26 @@ default List<Task<?>> getTasks(User user, Map<String, Pattern> filters, WebQuery
                         LinkedHashMap::new
                 ))
                 .values());
+    }
+
+    default List<Task<?>> getTasks(User user, Map<String, Pattern> filters) throws IOException {
```

Comment:
I would rather define a functional interface [like this](https://github.com/ICIJ/datashare/compare/main...spike/task-args-and-result-serialization#diff-5f6c39623b76aa237c3a7bdff7a504a4d58d27fcbc1962b3a633cdd34c271032R1) and then define this api which is more generic
```java
default List<Task<?>> getTasks(User user, TaskFilter filter)
```
and passe the hash map filter as a lambda